rekordcrate/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "rekordcrate"
+version = "0.1.1"
+authors = ["Jan Holthuis <jan.holthuis@ruhr-uni-bochum.de>"]
+description = "Library for parsing Pioneer Rekordbox device exports"
+readme = "README.md"
+repository = "https://github.com/Holzhaus/rekordcrate"
+license = "MPL-2.0"
+keywords = ["rekordbox", "dj", "export", "database", "parser"]
+categories = ["parser-implementations"]
+edition = "2021"
+exclude = [".*"]
+
+[dependencies]
+nom = "7"
+rekordcrate_derive = { path = "../rekordcrate_derive" }
```

Comment:
Will this relative path work with `crates.io`? I guess it doesn't...

Reply:
https://github.com/serde-rs/serde/blob/master/serde/Cargo.toml